source/tutorials/nix-language.md

```diff
@@ -2037,7 +2037,14 @@ Explanation:
 ### Get things done
 
 - [](declarative-reproducible-envs) – create reproducible shell environments from a Nix file
-- [Garbage Collection](https://nix.dev/manual/nix/2.18/package-management/garbage-collection.html) – remove unused build results from the Nix store
+- [](./packaging-existing-software.md) – make more software available through Nix with one command
```

Comment:
Why mention `with one command` ?

It read and understand this as if I can use one command to package existing software 🤔

Reply:
```suggestion
- [](./packaging-existing-software.md) – make more software available through Nix
```